### PR TITLE
Support :w and :q commands

### PR DESCRIPTION
**Pull request by aminroosta**
`:` will open Command Palette.
`:w` and `:q` are supported.

### CODE REVIEW

**Comment by smeggingsmegger** [comment]
👍 I'd love to see support for :w and :q in particular.

File: package.json [hunk 1]
```diff
@@ -33,6 +33,10 @@
     },
     "main": "./out/src/extension",
     "contributes": {
+        "commands": [
+            { "title": "w", "command": "workbench.action.files.save" },
+            { "title": "q", "command": "workbench.action.closeActiveEditor" }
+        ],
```
Comment (aioutecism) [inline package.json]: I think we should implement our own class to handle vim command using `showInputBox`.
Register commands to vscode's command palette will confuse normal usage of it.

**Comment by aioutecism** [comment]
However, I think `:w` and `:q` is a step back comparing to `cmd+s` and `cmd+w`.
Default shortcuts are easy to press and faster(2 keys comparing to 3).
And, user can always customize keybindings if they're not comfortable with the defaults.

**Comment by smeggingsmegger** [comment]
Faster unless you've used Vim for 20+ years. Don't discount muscle memory.

**Comment by aminroosta** [comment]
@smeggingsmegger  You might [consider](https://github.com/VSCodeVim/Vim/blob/master/ROADMAP.md) using https://github.com/VSCodeVim/Vim.

**Comment by aioutecism** [comment]
@smeggingsmegger Sorry to hear that. But this project is trying to take the good parts of Vim into VSCode, not mimic all Vim behaviors. If that is the case, Vim with those awesome plugins may be a better choice.

**Comment by smeggingsmegger** [comment]
No worries. I understand completely @aioutecism. I was just challenging your assertion that CMD-S was *necessarily* faster. I get why you don't want to add it to this particular project. I don't mind overmuch. I can write my own companion plugin if I want it that badly.

**Comment by aioutecism** [comment]
@smeggingsmegger 😄

**Pull request closed without merging**